package.json

```diff
@@ -17,18 +17,14 @@
     "@18f/stylelint-rules": "^2.0.0",
     "axios": "^0.18.0",
     "axios-mock-adapter": "^1.15.0",
-    "babel-core": "^6.26.0",
+    "babel-core": "^6.26.3",
     "babel-jest": "^17.0.0",
-    "babel-loader": "^7.1.2",
-    "babel-plugin-add-module-exports": "^0.2.1",
-    "babel-plugin-react-html-attrs": "^2.0.0",
-    "babel-plugin-transform-async-to-generator": "^6.16.0",
-    "babel-plugin-transform-class-properties": "^6.18.0",
-    "babel-plugin-transform-decorators-legacy": "^1.3.4",
+    "babel-loader": "^7.1.4",
     "babel-polyfill": "^6.26.0",
     "babel-preset-es2015": "^6.18.0",
     "babel-preset-react": "^6.16.0",
-    "babel-preset-stage-0": "^6.16.0",
+    "babel-preset-stage-2": "^6.24.1",
```

Comment:
Not sure why it was loading `stage-0` before, since the [`.babelrc`](https://github.com/18F/e-QIP-prototype/blob/master/.babelrc) has `stage-2` specified.